### PR TITLE
Roc plotly

### PR DESCRIPTION
**Pull request by JKL98ISR**

### CODE REVIEW

**Review by ItayGabbay — APPROVED** [review]

File: deepchecks/checks/performance/roc_report.py [hunk 1]
```diff
@@ -57,45 +57,55 @@ def _roc_report(self, dataset: Dataset, model):
...
         multi_y = (np.array(ds_y)[:, None] == np.unique(ds_y)).astype(int)
-        n_classes = len(dataset.classes)
         y_pred_prob = model.predict_proba(ds_x)
 
         fpr = {}
         tpr = {}
         roc_auc = {}
-        for i in range(n_classes):
+        for i, _ in enumerate(dataset_classes):
             if i in self.excluded_classes:
```
Comment (matanper) [inline deepchecks/checks/performance/roc_report.py:68]: I think we need to change this to work directly on the class, not the class index

Comment (JKL98ISR) [reply deepchecks/checks/performance/roc_report.py:68]: why?

Comment (JKL98ISR) [reply deepchecks/checks/performance/roc_report.py:68]: I'm using an array

Comment (matanper) [reply deepchecks/checks/performance/roc_report.py:68]: I'm not talking about the array, if the labels are 'a', 'b', 'c' the user will exclude [a,b] and not [1,2]

Comment (JKL98ISR) [reply deepchecks/checks/performance/roc_report.py:68]: oh yeah forgot about the excluded classes

Comment (JKL98ISR) [reply deepchecks/checks/performance/roc_report.py]: changed it so it'll skip by name, I still need the relative index because of the prediction series

**Pull request merged**